### PR TITLE
fixed bugs found while creating new officers

### PR DESCRIPTION
**Pull request by justend29**
made manage officers pretty, too

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -60,7 +64,17 @@ class _ParkrAppState extends State<ParkrApp> {
...
+      home: FutureBuilder<Widget>(
+          future: _buildStartPage(context),
+          builder: (BuildContext ctx, AsyncSnapshot<Widget> snapshot) {
+            if (snapshot.hasError) {
+              print(snapshot.error);
+              return const Text(
+                  "An internal Parkr error occurred. Please reload the application.");
+            }
+            return snapshot.hasData ? snapshot.data as Widget : const Text(
+                "Jesus Loves You");
```
Comment (StephenCole19) [inline lib/main.dart:76]: yeet

File: lib/widgets/loginform.dart [hunk 2]
```diff
@@ -103,16 +122,11 @@ class _LoginFormState extends State<LoginForm> {
...
-                            login(context),
-                            "Logging In...",
-                            null,
-                            "Failed to log ${emailCtrl.text} in") !=
-                        null) {
+                    if (await login(context) != null) {
                       Navigator.pushNamedAndRemoveUntil(
                           context, "home", (_) => false);
+                    } else {
+                      print("LITERALLY FUCK YOU");
```
Comment (StephenCole19) [inline lib/widgets/loginform.dart:129]: yeet

**Review by StephenCole19 — CHANGES_REQUESTED** [review]
Please fix *interesting prints*

File: lib/main.dart (hunk 1 above)
Comment (justend29) [reply lib/main.dart:76]: this has to actually stay

File: lib/widgets/loginform.dart (hunk 2 above)
Comment (justend29) [reply lib/widgets/loginform.dart:129]: thank you for catching this

**Review by StephenCole19 — APPROVED** [review]

**Pull request merged**